docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -14,28 +14,41 @@ Deploying databases along with Strapi is covered in the [databases guide](/devel
 
 ## General guidelines
 
-::: prerequisites
+### Prerequisites
```

Comment:
We have reserved "prerequisites" for the callout and it is not being used otherwise for headers. I can try to reword it if you want.

Reply:
I will rename the section and keep the table. Otherwise, I think it is fine and an improvement.